Rakefile

```diff
@@ -158,35 +158,7 @@ def upload_widget(version)
 end
 
 def upload_bower(version)
-  submodule = "cd submodules/uploadcare-bower &&"
-
-  # Check if such verion already exists
-  print `#{submodule} git reset --hard && git checkout master && git clean -fdx && git pull`
-  tag = `#{submodule} git tag -l #{version}`.strip
-  if not tag.empty?
-      print "ERROR: The version already exists: <#{tag}>\n"
-      return
-  end
-
-  cp = lambda do |name|
-    FileUtils.cp "pkg/#{version}/#{name}", "submodules/uploadcare-bower/#{name}"
-    `#{submodule} git add "#{name}"`
-  end
-
-  # Copy files from release
-  PACKAGES.each do |package|
-    cp.call("#{package}.js")
-    cp.call("#{package}.min.js")
-  end
-
-  # Update version number in boser.json
-  `#{submodule} sed -i -e 's/^  "version": "[^"]*"/  "version": "#{version}"/g' bower.json`
-  `#{submodule} git add bower.json`
-
-  # Commit, create a tag, and push
-  `#{submodule} git commit -m"New widget release: #{version}"`
-  `#{submodule} git tag #{version}`
-  `#{submodule} git push origin master && git push --tags`
+  print "bower obsolete, use npm instead\n"
```

Comment:
I think we can drop this method along with the build task

Reply:
I will drop all build tasks soon 🤪

Reply:
@jeetiss oga 🦀